pages/common/meteor.md

```diff
@@ -0,0 +1,27 @@
+# meteor
+
+> Full-stack javascript platform for building web applications.
+
+- Run a meteor project from its root directory in development mode:
+
+`meteor`
+
+- Create a project under the given directory:
+
+`meteor create {{path/to/directory}}`
+
+- Display the list of packages your project is currently using:
```

Comment:
I'd rather avoid the "you" (yet another thing that could be in the guidelines, although we must take care not to overdo it)

How about: "Display the list of packages the project is currently using" (or "a project"), "Display the list of packages the current project is using", or "Display the list of packages used in the current project"?